### ISSUE TITLE
Ecto instrumentation crashes with telemetry 0.4 and latest ecto_sql

### ISSUE THREAD

**Issue by lukaszsamson** [appsignal/appsignal-elixir#457]
appsignal: 1.9.1
ecto: 3.0.7
ecto_sql: github master 5e78291dbc8c0c6d249d8ad7150c120a4a836c59
telemetry: 0.4.0
```
[error] Handler "appsignal-ecto" has failed and has been detached. Exception: class=:error reason={:badkey, :queue_time, %{params: [], query: "(omitted)", repo: MyApp.Repo, result: :ok, source: "my_entities", type: :ecto_sql_query}} stacktrace=[{Appsignal.Ecto, :log, 1, [file: 'lib/appsignal/ecto.ex', line: 45]}, {:telemetry, :"-execute/3-fun-0-", 4, [file: '/myapp/deps/telemetry/src/telemetry.erl', line: 120]}, {:lists, :foreach, 2, [file: 'lists.erl', line: 1338]}, {Ecto.Adapters.SQL, :log, 4, [file: 'lib/ecto/adapters/sql.ex', line: 783]}, {DBConnection, :log, 5, [file: 'lib/db_connection.ex', line: 1317]}, {Ecto.Adapters.SQL, :execute!, 4, [file: 'lib/ecto/adapters/sql.ex', line: 548]}, {Ecto.Adapters.SQL, :execute, 5, [file: 'lib/ecto/adapters/sql.ex', line: 540]}, {Ecto.Repo.Queryable, :execute, 4, [file: 'lib/ecto/repo/queryable.ex', line: 147]}, {Ecto.Repo.Queryable, :all, 3, [file: 'lib/ecto/repo/queryable.ex', line: 18]}, {MyApp.MyModule, :my_function, 0, [file: 'lib/myapp/my_module.ex', line: 48]}]
```

**Comment by jeffkreeftmeijer** [comment]
Hey @lukaszsamson,

Thanks for letting us know about this! The event durations changed in Telemetry 0.4, which required us to add a clause to match both. Could you try the [ecto-durations](https://github.com/appsignal/appsignal-elixir/tree/ecto-durations) branch to see if that resolves your issue?

Thanks!
-- Jeff

**Comment by lukaszsamson** [comment]
@jeffkreeftmeijer I confirm it fixes the issue

**Comment by tombruijn** [comment]
@jeffkreeftmeijer shall we make a PR for that branch then?

**Comment by tombruijn** [comment]
Fixed in #459

**Comment by jeffkreeftmeijer** [comment]
Hey @lukaszsamson,

I've just released version 1.9.4 of the AppSignal for Elixir integration, which includes this fix. Could you switch to that and let me know if that still fixes your issue?
